virtool/config/cli.py

```diff
@@ -78,12 +78,22 @@ def server():
 @postgres_connection_string_option
 @redis_connection_string_option
 @sentry_dsn_option
-def start_api_server(**kwargs):
+@click.option(
+    "--flags",
+    help="feature flag of the feature to enable",
+    type=click.Choice(
+        ['FF_ADMINISTRATOR_ROLES', 'FF_ML_MODELS', 'FF_SPACES'], case_sensitive=True
+    ),
```

Comment:
Try this:
```suggestion
    type=FeatureFlag
```

If this doesn't work:
1. Check if there is some other way to make `click` work with an enum.
2. Use a comprehension to iterate through the enum instead of hardcoding the list.